warehouse/templates/base.html

```diff
@@ -78,6 +78,14 @@
 
     <link rel="search" type="application/opensearchdescription+xml" title="PyPI" href="{{ request.route_path('opensearch.xml') }}">
 
+    {# This needs to be included first (in order to capture errors when
+     # executing other JS scripts. It also can't be async, because then order
+     # of execution is not guaranteed, and it can't be deferred, because most
+     # of our JS would then execute before it, and we might miss exceptions.
+     #}
+    {% if request.registry.settings.get("sentry.frontend_dsn") -%}
+    <script src="https://cdn.ravenjs.com/3.26.2/raven.min.js" crossorigin="anonymous"></script>
```

Comment:
can we add a [`integrity`](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) attribute here with the expected checksum? this is a lot more feasible with a versioned url, unlike our Google Analytics and Fastly Insights loads.